### ISSUE TITLE
AWS key authentication

### ISSUE THREAD

**Issue by jmayhugh** [BrianAltenhofel/search_api_elasticsearch#93]
Is there a way to use this module with AWS-hosted Elasticsearch using an auth key? It doesn't look like it from the interface, but I just wanted to confirm that I'm not missing something.

**Comment by MatWaller** [comment]
I don't believe so,  could you not add the ssh key to the owner/group running php?
